This patch adds "LibreDNS" as a DNS-over-HTTPS (DoH) provider to LibreWolf/Firefox
--- src/firefox-86.0/modules/libpref/init/all.js	2021-03-03 00:00:00 +0100
+++ src/firefox-86.0/modules/libpref/init/all.js	2021-03-03 00:00:00 +0100
@@ -4031,10 +4031,10 @@
 // 0 - default off, 1 - reserved/off, 2 - TRR first, 3 - TRR only, 4 - reserved/off, 5 off by choice
 pref("network.trr.mode", 0);
 // DNS-over-HTTP service to use, must be HTTPS://
-pref("network.trr.uri", "https://mozilla.cloudflare-dns.com/dns-query");
+pref("network.trr.uri", "https://doh.libredns.gr/ads");
 // List of DNS-over-HTTP resolver service providers. This pref populates the
 // drop-down list in the Network Settings dialog box in about:preferences.
-pref("network.trr.resolvers", "[{ \"name\": \"Cloudflare\", \"url\": \"https://mozilla.cloudflare-dns.com/dns-query\" },{ \"name\": \"NextDNS\", \"url\": \"https://firefox.dns.nextdns.io/\" }}]");
+pref("network.trr.resolvers", "[{ \"name\": \"Cloudflare\", \"url\": \"https://mozilla.cloudflare-dns.com/dns-query\" },{ \"name\": \"NextDNS\", \"url\": \"https://firefox.dns.nextdns.io/\" },{ \"name\": \"LibreDNS\", \"url\": \"https://doh.libredns.gr/ads\" }]");
 // credentials to pass to DOH end-point
 pref("network.trr.credentials", "");
 pref("network.trr.custom_uri", "");
